entrypoint.sh

```diff
@@ -20,14 +24,28 @@ javawrapperversion=$(curl https://repo1.maven.org/maven2/com/veracode/vosp/api/w
 
 echo "javawrapperversion: $javawrapperversion"
 
+# Building jar execution command
+veracodejavaapicmd='/usr/local/openjdk-8/bin/java -jar VeracodeJavaAPI.jar -action UploadAndScan -autoscan true'
+
+# if $var is set: add flag & value
+[ ! -z "$appname" ] && veracodejavaapicmd+=' -appname "$appname"'
+[ ! -z "$createprofile" ] && veracodejavaapicmd+=' -createprofile "$createprofile"'
+[ ! -z "$filepath" ] && veracodejavaapicmd+=' -filepath "$filepath"'
+[ ! -z "$version" ] && veracodejavaapicmd+=' -version "$version"'
+[ ! -z "$vid" ] && veracodejavaapicmd+=' -vid "$vid"'
+[ ! -z "$vkey" ] && veracodejavaapicmd+=' -vkey "$vkey"'
+[ ! -z "$sandboxname" ] && veracodejavaapicmd+=' -sandboxname "$sandboxname"'
```

Comment:
Dynamically building the veracode scanning command

This allows us to exclude flags if we don't want to use them